ang/test/civicase/contact/directives/contact-card.directive.spec.js

```diff
@@ -69,6 +69,20 @@
         });
       });
 
+      describe('when the display name does not have first name or last name but an email id either', function () {
+        beforeEach(function () {
+          spyOn(ContactsCache, 'add').and.returnValue($q.resolve(ContactsData.values[0]));
+          spyOn(ContactsCache, 'getCachedContact').and.returnValue({
+            display_name: 'John Doe'
+          });
+          compileDirective(true, ContactsData.values[0].id);
+        });
+
+        it('sets the initial of the name as the avatar', function () {
+          expect(element.isolateScope().contacts[0].avatar).toBe('JD');
+        });
+      });
```

Comment:
Looking at the logic implemented in `getInitials`, aren't we missing a test for contacts with a single display name?

Reply:
As discussed this is not needed.